contracts/Removal.sol

```diff
@@ -403,32 +436,6 @@ contract Removal is
     return super.supportsInterface(interfaceId);
   }
 
-  function isApprovedForAll(address account, address operator)
-    public
-    view
-    override
-    returns (bool)
-  {
-    return
-      account == address(_market) ||
```

Comment:
I am pretty certain that this stuff was accidentally included in my documentation PR when it had actually been removed from Jaycen's Removal contract cleanup PR.  we explicitly talked about not needing the market to have operator approval for any reason, and actually including this always-true check for the market being approved introduced a security vulnerability where someone random could call `withdraw` on the market contract and have the call succeed.